build-system/babel-plugins/babel-plugin-transform-stringish-literals/index.js

```diff
@@ -132,16 +127,20 @@ module.exports = function ({types: t}) {
           }
         }
 
-        if (conversions === 0) {
+        newQuasis = newQuasis.filter(Boolean);
```

Comment:
I'm also noticing an issue with our quasi collapsing. We're not properly escaping the expression string in the `raw` path: https://astexplorer.net/#/gist/ee7d0673c706c9176890bb7e61d1a1f7/f60a0a977bec8e49095a4c17a0c801421c3f0214